pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -43,7 +43,7 @@ const (
 
 	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
 	// Used to determine if CRD needs to be updated in cluster
-	CustomResourceDefinitionSchemaVersion = "1.15"
+	CustomResourceDefinitionSchemaVersion = "1.16"
```

Comment:
Maybe the schema version should actually be 1.15.1 instead? @aanm

Reply:
Given your approval + the later Cilium versions' CRD Schema versions being distinct and higher, I do not plan to change this. If you change your mind and think I should change it, please let me know.